integrations/helm/operator/operator.go

```diff
@@ -330,7 +333,11 @@ func (c *Controller) enqueueUpateJob(old, new interface{}) {
 
 	if diff := cmp.Diff(oldFhr.Spec, newFhr.Spec); diff != "" {
 		c.logger.Log("info", "UPGRADING release")
-		c.logger.Log("info", fmt.Sprintf("Custom Resource driven release upgrade, diff:\n%s", diff))
+		if c.logDiffs {
+			c.logger.Log("info", fmt.Sprintf("Custom Resource driven release upgrade, diff:\n%s", diff))
```

Comment:
Could be another key/value pair here? Something something structured logging

Reply:
I'm all for structured logging, that's actually what I tried first, but I didn't have much luck getting it to work.  I'll give it another shot.

Reply:
Whatever problem I had before is not present now.